docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md

```diff
@@ -70,16 +70,16 @@ $ export HostIP="<Master node IP>"
 >**NOTE:** This etcd instance is separate from the **etcd** deployed by Kubernetes/OpenShift and is **required** for managing the **HPE 3PAR Volume Plug-in for Docker**. We need to modify the default ports (**2379, 4001, 2380**) of the **new etcd** instance to prevent conflicts. This allows two instances of **etcd** to safely run in the environment.`
 
 ```yaml
-sudo docker run -d -v /usr/share/ca-certificates/:/etc/ssl/certs -p 40010:40010 \
--p 23800:23800 -p 23790:23790 \
+sudo docker run -d --restart unless-stopped -v /usr/share/ca-certificates/:/etc/ssl/certs -p 4001:4001 \
```

Comment:
Please do not change the ports in the md file as we are using different ports to avoid conflicts with k8s internal etcd.